### ISSUE TITLE
(H)ēsiodos et cetera... Gerard is not able to work (Search is not possible: Ajax error)

### ISSUE THREAD

**Issue by Hippodamie** [eXistSolutions/LGPN-ling#384]
Dear Magdalena,

Hi, how are you ?
The bug on Hesiodos was due to a change in the sub-window and several re-entering in the EDition window by both Gerard and I.
Between he and I, it seems that you fixed it, but I re-entered and... bug again.

We will have those times again, I think to modify the form of the label and the root in the sub-form window, perhaps for lots of names.

Is it possible that this kind of modifications do not generate bugs? It seems that this sub-form window is very sensitive and does not like at all modifications. But it is usual for us to need to make changes also in this field as our work progress.
So could it be possible it becomes not so sensible ?

Could you, please, also answer at my question Issue 382 about LABEL:
1. Is necessarily the label exactly corresponding to the root:
e.g. if the root is θε(ο/ᾱ), has the label to be θεοᾱ (with the same  letters)?
2. Diacritics like the macron on θεοᾱ are they ‘authorized’ by the system in the label, or do we have better to write:  θεοα without the macron of the Root?
3. Could we imagine to have as a label only : θε and as a root θε(ο/ᾱ)?
In other words, how did you plan/understand ‘label’? How do you and/or will you use it in the system? Does it only belong to the back office and was it conceived only to be sure not to disturb the machine with parasit signs as macron parentheses etc.?
Because, for us, if we write, e. g. θεοᾱ as a label it does not make any sense. So, do we have to understand that the label is only useful for the system and not for us?
Could you please re-explain it for us ?

Thanks for your answer.
Could you please help for Gerard to be able to work again? Many thanks in advance.
With all best wishes,
Sophie
